### PR TITLE
Player scene integration

### PR DESCRIPTION
**Pull request by UsingNamespaceNolan**
Created first city scene with a TileMap. Added Player moving and stationary animations. Added play button to main scene which navigates to first city scene. Changed the way Player moves: instead of altering global_position (altering this fucks with godots collision detection system), just set the "velocity" variable in _physics_process callback and call the built-in move_and_slide(). FWIW I think this is the way we should be moving all of the moving bodies, as you can just set the collision layers and collision masks and collision detection works seamlessly. I think I added a bunch of assets too

### CODE REVIEW

File: Temporal Mirage - Cybernetic Reclaim/Player/Character.gd [hunk 1]
```diff
@@ -170,3 +165,20 @@ func throw_grenade():
 
 func reload_grenades():
 	grenades = max_grenades
+	
+func _get_velocity_from_key_in():
+	#TODO separate setting of animation to separate class?
```
Comment (BradMcGlynnUWO) [inline Temporal Mirage - Cybernetic Reclaim/Player/Character.gd:170]: what do you mean by this? Have a separate class for all animation movements?

File: Temporal Mirage - Cybernetic Reclaim/BaseClasses/Characters/BaseCharacter.gd [hunk 2]
```diff
@@ -16,9 +16,6 @@ func take_damage(amount: int) -> void:
 
 func die() -> void:
 	queue_free() # Override this method to perform specific actions upon death.
-
-func move(direction: Vector2, delta: float) -> void:
```
Comment (BradMcGlynnUWO) [inline Temporal Mirage - Cybernetic Reclaim/BaseClasses/Characters/BaseCharacter.gd:20]: can the enemy characters still move with this change? I'm just looking at the code rn, I'll pull down the branch and check it out myself soon tho

Comment (BradMcGlynnUWO) [reply Temporal Mirage - Cybernetic Reclaim/BaseClasses/Characters/BaseCharacter.gd:20]: I added the movement for enemies so this point doesn't matter anymore

**Comment by BradMcGlynnUWO** [comment]
Looks fine to me, movement looks good plus the scene loads well

File: Temporal Mirage - Cybernetic Reclaim/BaseClasses/Characters/BaseCharacter.gd (hunk 2 above)
Comment (UsingNamespaceNolan) [reply Temporal Mirage - Cybernetic Reclaim/BaseClasses/Characters/BaseCharacter.gd:20]: min, thanks for taking care of it

File: Temporal Mirage - Cybernetic Reclaim/Player/Character.gd (hunk 1 above)
Comment (UsingNamespaceNolan) [reply Temporal Mirage - Cybernetic Reclaim/Player/Character.gd:171]: I meant a separate function, call it in the physics process to set the animation or something

**Pull request merged**